src/lib/CometVisuClient.js

```diff
@@ -175,6 +175,9 @@ define( ['jquery'], function( $ ) {
             this.readResendHeaderValues();
             session.update(data);
             this.retryCounter = 0;
+            if (!session.dataReceived) {
```

Comment:
Do we need here (and in the places below) really the if statement? Wouldn't it be sufficient to just set `session.dataReceived` unconditionally?